WordPress/WordPressTest/SiteCreation/SiteCreationNameTracksEventTests.swift

```diff
@@ -21,6 +21,47 @@ class SiteCreationNameTracksEventTests: XCTestCase {
         return siteIntentViewController
     }
 
+    class MockSiteNameStep: WizardStep {
+        weak var delegate: WizardDelegate?
+        private let creator: SiteCreator
+
+        var expectedProperty = ""
+
+        var content: UIViewController {
+            SiteNameViewController(siteNameViewFactory: makeSiteNameView) { [weak self] in
+                SiteCreationAnalyticsHelper.trackSiteNameSkipped()
```

Comment:
It can, if we want to use `MockSiteNameStep` only for `testSiteNameTracksEventFiresWhenEntered`, but it might not hurt to leave it there if we add future tests and we want to use this mock for them?